skietbaan_fe/src/bootstrap/NavbarMenuStyle.css

```diff
@@ -1,24 +1,29 @@
-@media only screen and (min-width: 580px){
+@media only screen and (min-width: 576px){
   .dropdown-menu.show{
     background-color: #F3F4F9;
     border: none;
     border: 1px solid  #F3F4F9;
     
   }
 
+  .profile-icon-grey{
+    height: 27px;
+    width: 28px;
+  }
+
   .leaderboard-icon-grey{
```

Comment:
These three classes are the same. use one and give it a more generic name

Reply:
used a generic className